cypress/support/fragments/circulation-log/searchPane.js

```diff
@@ -12,6 +13,17 @@ export default {
     ]);
   },
 
+  getLastWeekSearchParam() {
+    const today = new Date();
+    const lastWeek = new Date(today.getFullYear(), today.getMonth(), today.getDate() - 7);
+    return `${DateTools.padWithZero(lastWeek.getMonth() + 1)}/${DateTools.padWithZero(lastWeek.getDate())}/${lastWeek.getFullYear()}`;
+  },
+
+  getTodaySearchParam() {
```

Comment:
may be these methods should be moved to dateTools as common methods

Reply:
I thought about that, but this method returns value with custom formmating. I didn't see were this formating need except inventory search

Reply:
you can add methods related with circ logic and may be someone find it usable in another app and reuse it and rename it to more common meaning